ibm/mas_devops/roles/ocp_login/defaults/main.yml

```diff
@@ -28,3 +28,6 @@ supported_cluster_types:
   - aws
 
 k8s_auth_host: "{{ lookup('env', 'K8S_AUTH_HOST') | default('', True) }}"
+
+# Dualstack required?
+ipv6_test: "{{ lookup('env', 'IPV6_TEST') | default('false', true) }}"
```

Comment:
I am not sure that ipv6_test is the best name. For this role it would be enable_dualstack ? Also this var needs to be added to the docs https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/ocp_login/README.md?plain=1#L56

Reply:
I'm using ipv6_test because that's the flag's name in devIT's api to request a fyre cluster of this flavor.  Is that a good reason enough to keep the name ipv6_test? @whitfiea

Reply:
I think I'll call it enable_ipv6 then. Might not be dual stack in the future